dora/core/client/fs/src/test/java/alluxio/client/file/cache/DefaultPageMetaStoreTest.java

```diff
@@ -73,6 +73,15 @@ public void before() {
         MetricsSystem.METRIC_REGISTRY.getGauges().get(MetricKey.CLIENT_CACHE_PAGES.getName());
   }
 
+  @Test
+  public void commitFile() throws PageNotFoundException {
+    String newTempFile = "newTempFile";
+    PageId newTempPage = new PageId(newTempFile, 2L);
+    mMetaStore.addPage(mPage, mPageInfo);
+    mMetaStore.commitFile(mPage.getFileId(), newTempFile);
+    assertEquals(newTempFile, mMetaStore.getPageInfo(newTempPage).getPageId().getFileId());
```

Comment:
Are there more information you can assert on, other than just the file/page id? Are there anything like `file.isComplete`? Are there anythign like file length?